pom.xml

```diff
@@ -45,6 +45,13 @@
             <version>8.0.17</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.4.Final</version>
+        </dependency>
+
```

Comment:
@RomanMinevich two empty lines are not required. Please remove one.

Reply:
done